### DISCUSSION TITLE
The latest release broke the geotiff rendering

### DISCUSSION THREAD

**Discussion by jmlord** [developmentseed/titiler#991] [Q&A] [answered]
#### Problem description

With the latest release only, we have a 404 error when loading the cog with leaflet. Here is the log from the output console:
```
GET
http://localhost/tiler/cog/tiles/3/2/2?url=/output/data/loadFromStac/23020bc95fbf990fa92e374996903ba9/esacci-lc-2020_6efe64f5a2020-01-01.tif&rescale=11,210&colormap_name=hot
[HTTP/1.1 404 Not Found 2ms]
```

We are using the titiler docker, with a bind mount to a file system that hosts the geotiffs. Hence from tititler point of view, the tiffs are loaded from the same machine.
``` dockerfile
  tiler:
    container_name: biab-tiler
    image: ghcr.io/developmentseed/titiler:latest
    volumes:
      - ${PIPELINE_REPO_PATH}/output:/output:ro
      - ${PIPELINE_REPO_PATH}/userdata:/userdata:ro
    user: ${MY_UID}:${DOCKER_GID}
    environment:
      PORT: 8000
      WORKERS_PER_CORE: 1
      TITILER_API_CORS_ORIGINS: "*"
    extra_hosts:
      - "host.docker.internal:host-gateway"
```

#### Expected Output
No 404 error, we see the texture.

#### Environment Information
This is the problematic version:
https://github.com/developmentseed/titiler/pkgs/container/titiler/278530292?tag=latest

Reverting to 0.18.9 works as a workaround.

**Comment by vincentsarago** [comment, marked as answer]
Hi @jmlord, we're working on the next version of titiler, there will be a lot of breaking changes as mentioned in: https://github.com/developmentseed/titiler/blob/main/CHANGES.md#0190-tbd

titiler and other dependencies (e.g rio-tiler) are moving pieces. To make things better we have to introduce breaking changes. User should `pin` version of titiler python module of docker image 🙏

**Reply by jmlord** [reply]
Oh, right, I see the change there. Thanks.

I had indeed pinned to 0.18.9 when it occurred. @glaroc you might want to pin your titiler instances as well (or update the clients with the new API).
